consensus/hotstuff/verification/combined_verifier_v2.go

```diff
@@ -55,44 +57,44 @@ func (c *CombinedVerifier) VerifyVote(signer *flow.Identity, sigData []byte, blo
 	// TODO: to be replaced by packer
 	stakingSig, beaconShare, err := signature.DecodeDoubleSig(sigData)
 	if err != nil {
-		return false, fmt.Errorf("could not split signature: %w", modulesig.ErrInvalidFormat)
+		return fmt.Errorf("could not split signature: %w", modulesig.ErrInvalidFormat)
 	}
 
 	dkg, err := c.committee.DKG(block.BlockID)
 	if err != nil {
-		return false, fmt.Errorf("could not get dkg: %w", err)
+		return fmt.Errorf("could not get dkg: %w", err)
 	}
 
 	// verify each signature against the message
 	// TODO: check if using batch verification is faster (should be yes)
 	stakingValid, err := signer.StakingPubKey.Verify(stakingSig, msg, c.stakingHasher)
 	if err != nil {
-		return false, fmt.Errorf("internal error while verifying staking signature: %w", err)
+		return fmt.Errorf("internal error while verifying staking signature for %x: %w", signer.NodeID, err)
 	}
 	if !stakingValid {
-		return false, fmt.Errorf("invalid staking sig")
+		return fmt.Errorf("invalid staking sig for block %v: %w", block.BlockID, model.ErrInvalidSignature)
 	}
 
 	// there is no beacon share, no need to verify it
 	if beaconShare == nil {
-		return true, nil
+		return nil
 	}
 
 	// if there is beacon share, there must be beacon public key
 	beaconPubKey, err := dkg.KeyShare(signer.NodeID)
 	if err != nil {
-		return false, fmt.Errorf("could not get random beacon key share for %x: %w", signer.NodeID, err)
+		return fmt.Errorf("could not get random beacon key share for %x: %w", signer.NodeID, err)
 	}
 
 	beaconValid, err := beaconPubKey.Verify(beaconShare, msg, c.beaconHasher)
 	if err != nil {
-		return false, fmt.Errorf("internal error while verifying beacon signature: %w", err)
+		return fmt.Errorf("internal error while verifying beacon signature: %w", err)
 	}
 
```

Comment:
previous and following lines both handle the output of `Verify` call. In my opinion, separating them just bloats the code. 
```suggestion
```